broadbean/blueprint.py

```diff
@@ -530,6 +720,16 @@ def setSR(self, SR):
         """
         self._SR = SR
 
+    def set_sample_rate(self, SR: int | float) -> None:
+        """
+        Set the associated sample rate
+
+        Args:
+            SR (Union[int, float]): The sample rate in Sa/s.
```

Comment:
no need for the type in the docstring because it's in the signature already
```suggestion
            SR: The sample rate in Sa/s.
```

Reply:
Good call, I'll remove these legacy types in the doc string.